HWTS-3/task-1.ts

```diff
@@ -0,0 +1,23 @@
+interface ivehicle {
```

Comment:
```suggestion
interface IVehicle {
```
А вот интерфейсы с большой буквыназываются